spec/models/loader_orchestrate_spec.rb

```diff
@@ -65,58 +70,51 @@ def verify_data path
 
     context "with attempted policy update" do
       it "applies the policy update" do
-        load_policy_update 'simple.yml'
+        replace_policy_with 'simple.yml'
         verify_data 'updated/simple.txt'
       end
       it "creates a host factory" do
-        load_policy_update 'host_factory.yml'
+        replace_policy_with 'host_factory.yml'
         verify_data 'updated/host_factory.txt'
       end
       it "adds a layer to a host factory" do
-        load_policy_update 'host_factory.yml'
-        load_policy_update 'host_factory_new_layer.yml'
+        replace_policy_with 'host_factory.yml'
+        replace_policy_with 'host_factory_new_layer.yml'
         verify_data 'updated/host_factory_new_layer.txt'
       end
       it "removes a layer from a host factory" do
-        load_policy_update 'host_factory_new_layer.yml'
-        load_policy_update 'host_factory.yml'
+        replace_policy_with 'host_factory_new_layer.yml'
+        replace_policy_with 'host_factory.yml'
         verify_data 'updated/host_factory.txt'
       end
       it "doesn't affect a record in a different account" do
         Role.where(Sequel.function("account", :role_id) => 'acct1').delete
         Role.create(role_id: 'acct1:group:the-policy/group-a')
-        load_policy_update 'simple.yml'
+        replace_policy_with 'simple.yml'
```

Comment:
I'd use policy files for both accounts, rather than dropping down into model level logic.

Reply:
At first glance I was not able to find a policy which will be able to complete the following step `Role.where(Sequel.function("account", :role_id) => 'acct1').delete`. If there isn't such a policy, does it make sense to switch the role creation to a policy?